src/index.android.js

```diff
@@ -0,0 +1,22 @@
+import { NativeModules } from "react-native";
+
+const { NavigationBarColor } = NativeModules;
+
+const changeNavigationBarColor = (
+  color = String,
+  light = false,
+  animated = true
+) => {
+  const LightNav = light ? true : false;
+  NavigationBarColor.changeNavigationBarColor(color, LightNav, animated);
+};
+
+const hideNavigationBar = () => {
+  return NavigationBarColor.hideNavigationBar();
+};
```

Comment:
Can use implicit return here. Same for the `showNavigationBar` function.
```suggestion
const hideNavigationBar = () => NavigationBarColor.hideNavigationBar();
```